roles/ansible-keylime-tpm20/tasks/keylime.yml

```diff
@@ -15,6 +15,24 @@
     mode: 0644
     remote_src: yes
 
+- name: Create /etc/keylime for new-format config files
+  ansible.builtin.file:
+    path: /etc/keylime
+    state: directory
+    mode: 0755
+
+- name: Convert keylime.conf to new config format
+  shell: "/usr/bin/python3 convert_config.py"
+  args:
+    chdir: /root/keylime/scripts
+
+- name: Copy converted config files to /etc/keylime
+  copy:
+    src: /root/keylime/scripts/agent.conf /root/keylime/scripts/verifier.conf /root/keylime/scripts/tenant.conf /root/keylime/scripts/registrar.conf /root/keylime/scripts/ca.conf /root/keylime/scripts/logging.conf
```

Comment:
I don't believe that `src` can be multiple files like this. You will need to loop over the filenames and use them as a variable in the task. See https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#iterating-over-a-simple-list

Reply:
@mpeters Fixed, thanks for the pointer.